docs/containers.conf.5.md

```diff
@@ -405,7 +405,7 @@ if you want to set environment variables for the container.
 
 Define where event logs will be stored, when events_logger is "file".
 
-**events_logfile_max_size**=0
+**events_logfile_max_size**=1048576
```

Comment:
Could we make this human understandable 1mb?

Reply:
Great idea. We can use https://pkg.go.dev/github.com/docker/go-units?utm_source=godoc#FromHumanSize but have to change the type from uint64 to string.

I think that's perfectly fine since we didn't release it yet.